index.html

```diff
@@ -0,0 +1,158 @@
+        <span class="bar"></span>
+        <span class="bar"></span>
+        <span class="bar"></span>
+      </div>
+    </nav>
+  </header>
+  <!-- the Home page -->
+  <section id="home">
+    <div class="main-content">
+      <div class="headline">
+        <h2 class="heading1">"Hello! Sharing World"</h2>
+        <h1 class="main-heading">HopeGrower Gold Global Conference 2023</h1>
+      </div>
+      <p class="main-detail">A joyful celebration believing in the openness and sharing, believing in "Food is the New
+        Gold" creating a positive
+        change with people from over 100 countries is taking place in November, Pakistan</p>
+      <div class="date-venue">
+        <h2 class="dates">2023.11.14(THU) ~ 15(FRI)</h2>
+        <p class="venue">@ National Museum of Pakistan, Art Center Iqbal & more</p>
+      </div>
+    </div>
+  </section>
+  <!-- the Program page -->
+  <section id="program">
+    <div>
+      <h3 class="program-headline">Main Program</h3>
+      <div class="underline"></div>
+    </div>
+    <div class="program-container">
+      <div class="main-program">
+        <div><img src="/img/program_icon_01.png" alt=""></div>
+        <h3 class="program-subheading">
+          Lecture
+        </h3>
+        <p class="program-info">
+          Listen to the speakers from various countries about the message of farming and food.
+        </p>
+      </div>
+      <div class="main-program">
+        <div><img src="/img/program_icon_02.png" alt=""></div>
+        <h3 class="program-subheading">
+          HG Exhibition
+        </h3>
+        <p class="program-info">
+          Appreciate various creations applying CC license of artists, organized from Art Center Iqbal
+        </p>
+      </div>
+      <div class="main-program">
+        <div><img src="/img/program_icon_03.png" alt=""></div>
+        <h3 class="program-subheading">
+          Forum
+        </h3>
+        <p class="program-info">
+          Have the time to share your thoughts and opinions with experts for each topic.
+        </p>
+      </div>
+      <div class="main-program">
+        <div><img src="/img/program_icon_04.png" alt=""></div>
+        <h3 class="program-subheading">
+          Workshop
+        </h3>
+        <p class="program-info">
+          Try creating your own work using open source license rather than just watching at it.
+        </p>
+      </div>
+      <div class="main-program">
+        <div><img src="/img/program_icon_05.png" alt=""></div>
+        <h3 class="program-subheading">
+          HG Ignite
+        </h3>
+        <p class="program-info">
+          Get opportunities to network with HG affiliates around the world, also after the summit.
+        </p>
+      </div>
+    </div>
+    <button type="button" class="join-btn">Join HG Global Summit 2023</button>
+  </section>
+  <!-- the Speakers page -->
+  <section><h3 class="speaker-headline">Featured Speakers</h3></section>
+  <div id="speakers">    
+    
```

Comment:
- [ ] in the mobile version, only two speakers are visible unless you want to implement the see more functionality. if not kindly follow the design.
yours
![Screenshot (1084)](https://github.com/Mike111222/CapstoneProject/assets/65298048/5a18b34e-ac09-461e-925f-69798237ed1d)
main design
![Screenshot (1085)](https://github.com/Mike111222/CapstoneProject/assets/65298048/81e07c47-e630-4898-b33b-12a54a17567c)

Reply:
I have implemented the suggested changes with JavaScript sir. Thanks for the kind inputs.